editor/src/components/editor/store/store-hook.ts

```diff
@@ -28,7 +28,7 @@ export const useEditorState = <U>(
   if (context == null) {
     throw new Error('useStore is missing from editor context')
   }
-  return context.useStore(wrappedSelector, equalityFn)
+  return context.useStore(wrappedSelector, equalityFn as EqualityChecker<U>)
```

Comment:
This looks like a dubious lie, given that the second parameter of `EqualityChecker` is `unknown`

Reply:
in the case of our useEditorState selectors, since the selectors are enforced by typescript to always return `U` it makes our lives easier. if you manually write an equality function, and the second type is `unknown`, that equality function you are writing probably for performance reasons will have to spend time and effort on verifying that the type is indeed `U` even though typescript already did that for us.